ott/geometry/pointcloud.py

```diff
@@ -329,12 +323,20 @@ def transport_from_scalings(
     transport = jax.vmap(
         _transport_from_scalings_xy,
         in_axes=[
-            None, 0, None, self._axis_norm, None, 0, None, None, None, None
+            None,
```

Comment:
I like this formatting better than the one above on line 311 because it allows commenting the dimensions. I'll try provide an example.

Reply:
Yes, although here I must admit this is auto-formatted :)